### ISSUE TITLE
Dropdown component lazyload event callback is missing

### ISSUE THREAD

**Issue by et84121** [primefaces/primevue#2046]
**I'm submitting a ...**  (check one with "x")

[ x ] bug report

**Demo**
https://primefaces.org/primevue/showcase/#/dropdown
Virtual Scroll (1000 Items) and Lazy

**Current behavior**

```
<Dropdown v-model="selectedItem2" :options="lazyItems" optionLabel="label" optionValue="value" :virtualScrollerOptions="{ lazy: true, onLazyLoad: onLazyLoad, itemSize: 31, showLoader: true, loading: loading, delay: 250 }" placeholder="Select Item">
```
onLazyLoad callback have been removed 

**Expected behavior**

LazyLoad event should work

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Vue version:** 3.X

* **PrimeVue version:** 3.11.0 

* **Browser:** [all]

**Comment by mauritsderoover** [comment]
Is there already a timetable with respect to this issue?

**Comment by tugcekucukoglu** [comment]
In showcase demo onLazyLoad method still working on the lazy mode of dropdown. What exactly you mean `onLazyLoad callback have been removed` ?

**Comment by Mettbrot** [comment]
For me, the showcase does not work either. "Virtual Scroll (1000 Items) and Lazy" cannot load items past Item 34. It is just blank after the animation disappears. Firefox 96.0.3

**Comment by et84121** [comment]
@tugcekucukoglu 
"onLazyLoad callback have been removed" means according to 
https://github.com/primefaces/primevue/blob/e4789c9a73af469d72401aca8ff0158bee934a0e/src/components/dropdown/Dropdown.d.ts#L165
there is no onLazyload callback in 
https://github.com/primefaces/primevue/blob/e4789c9a73af469d72401aca8ff0158bee934a0e/src/components/virtualscroller/VirtualScroller.d.ts#L85-L156

But there is a 'lazy-load' event in
https://github.com/primefaces/primevue/blob/e4789c9a73af469d72401aca8ff0158bee934a0e/src/components/virtualscroller/VirtualScroller.d.ts#L252-L273

- - - - - - 

https://github.com/primefaces/primevue/blob/e4789c9a73af469d72401aca8ff0158bee934a0e/src/views/dropdown/DropdownDemo.vue#L55

https://github.com/primefaces/primevue/blob/e4789c9a73af469d72401aca8ff0158bee934a0e/src/views/virtualscroller/VirtualScrollerDemo.vue#L112-L116

**Comment by mauritsderoover** [comment]
Hi, 

I did some more research on this issue (that I does not seem to load past 34 elements). 

I saw that the VirtualScroller has two different slots (a content slot and a item slot) and that the Dropdown uses the content slot to customize the virtual scroller.  However, using the content slot the dropdown does not get past 34 elements. 

When I tried rewriting the dropdown with the virtual scroller using the item slot, then it worked perfectly. 

I am using: 
Browser: Edge, Firefox and Chrome
Vue: 3.2.27
PrimeVue: 3.11.0
